tests/unit_tests/simpll/DifferentialFunctionComparatorTest.cpp

```diff
@@ -314,6 +330,75 @@ class DifferentialFunctionComparatorTest : public ::testing::Test {
     }
 };
 
+/// Test fixture for testing DifferentialFunctionComparator
+/// for tests written in LLVM IR.
+class DFCLlvmIrTest : public DifferentialFunctionComparatorTest {
+  public:
+    std::string errorMsg = "";
+
+    DFCLlvmIrTest() {}
+
+    /// Tries to parse LLVM IR from string and prepares DFC for testing,
+    /// if parsing was unsuccessful returns false and the errorMsg is set.
+    bool prepare(const char *left, const char *right) {
+        ModL = stringToModule(left, CtxL);
+        if (ModL == nullptr) {
+            errorMsg = "Error while parsing left LLVM IR " + errorMsg;
+            return false;
+        }
+        ModR = stringToModule(right, CtxR);
+        if (ModR == nullptr) {
+            errorMsg = "Error while parsing right LLVM IR " + errorMsg;
+            return false;
+        }
+        FL = getFunByName(*ModL, "f");
```

Comment:
There's `llvm::Module::getFunction(StringRef Name)`, could we use it?

Reply:
Yes, thanks.